cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -197,7 +197,7 @@ def _create_validation_classes(
         json_fields: list[str] = []
         for prop_name, prop in view.properties.items():
             if isinstance(prop, dm.EdgeConnection):
-                edge_by_property[prop.type.external_id] = prop_name, prop
```

Comment:
^ the above produce major bug as the property name is not edge type but property in view that is of type edge

Reply:
Good catch.